skorch/tests/test_net.py

```diff
@@ -1829,6 +1829,19 @@ def check_grad(*args, loss, training, **kwargs):
         net = net_cls(module_cls, max_epochs=1, callbacks=[mock_cb])
         net.fit(*data)
 
+    def test_callback_on_grad_computed(self, net_cls, module_cls, data):
```

Comment:
Could you shortly explain why you added this test?

Reply:
This PR changes the `self.notify('on_grad_computed',  ...)` to use the `TeeGenerator`. This test makes sure that the named_parameters are still correctly passed to the callback.